timer.html

```diff
@@ -6,17 +6,18 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
 </head>
+
 <body>
     <div class="timer">
-        <span id="timerDisplay">0</span>
+        <span id="timer_display">0</span>
         <div>
             <button onclick="start()">Start</button>
             <button onclick="stop()">Stop</button>
             <button onclick="pause()">Pause</button>
             <button onclick="resume()">Resume</button>
         </div>
     </div>
+
 <script src="time.js"></script>
```

Comment:
I suggest make this code aligned with the div elements and not aligned with the body element.

https://github.com/PUP-BSIT/exercise-12-barrantes-sofiaanne/blob/8ce52c91caff496030a1154fe93dad0866e110e2/timer.html#L21